src/main/kotlin/step2/StringCalculator.kt

```diff
@@ -0,0 +1,40 @@
+package step2
+
+class StringCalculator {
+    companion object {
```

Comment:
내부 매서드들이 모두 companion object 안에 들어있고, 모두 순수 함수여서 object 키워드를 사용해도 좋아 보입니다.
```suggestion
object StringCalculator {
    ...
```

Reply:
이 방법을 모르고 있었네요 적용했습니다 👍